.github/workflows/test.yml

```diff
@@ -39,10 +39,19 @@ jobs:
         name: setup terraform
         uses: hashicorp/setup-terraform@v1  # sets up Terraform CLI in your GitHub Actions workflow
         with:
-          terraform_version: 0.13.0
+          terraform_version: 0.13.1
 
       - name: Run Test
         working-directory: test
         run: go test -v ./...
         env:
           IC_API_KEY: ${{ secrets.ACCESS_KEY }}
+
+      - uses: 8398a7/action-slack@v2
```

Comment:
looks like writing test results into slack channel is not working ... can you just confirm once